### ISSUE TITLE
Building translations AAR library

### ISSUE THREAD

**Issue by gkua** [wireapp/wire-android#1346]
## The problem
Can't seem to find a way to build AAR library for translations

## Environment
Windows 10, 64bit

## Details
`./gradlew assembledebug` and `gradlew assembledebug`

## Link to Debug logs

None

**Comment by someoneEsle** [comment]
Have you tried ./gradlew assembleDevDebug ?

**Comment by gkua** [comment]
Yeah, it seems to be not finding gradlew command - but when using gradlew command within wire-android it works perfectly, so i’m not sure how to fix this

**Comment by davidaxing** [comment]
How to build

**Comment by davidaxing** [comment]
How to build
